projects/baby_driver/scripts/can_util.py

```diff
@@ -183,3 +183,35 @@ def next_message(
         ))
 
     return msg
+
+def can_pack(data_list):
+    """
+    Converts list of tuples in form ((int) val, (int) len_in_bytes)
+            and combines them into a bytearray rendition
```

Comment:
Make sure you specify the endianness of the output bytes representing each `val`; after these changes it'll be little-endian order. You also might want an example here, and explain more that each `val` is broken into `len_in_bytes` bytes in little-endian order, then the bytes are concatenated in a `bytearray`.